news/_posts/2019-05-29-severity-classification.md

```diff
@@ -0,0 +1,103 @@
+
+The first thing to learn is that severity is to be distinguished from priority. Indeed, severity is used to measure the negative impact that a bug has on a system, a feature, a component or on the project development. It is usually defined by the QA team. As for the priority, it is used to organize all the tasks (bugs, improvements, features, technical tasks) that have to be done in order to meet the project’s deadlines. 
+
+
+## Issue Severity Criteria
+
+When a new issue is created, the first step is to understand what the problem is and then reproduce it. Once that work is done, the second step is to define the severity of that bug.
+
+Four severity levels have been defined, as explained below.
+
+
+### Critical
+
+The bug affects critical functionality or critical data and there is no workaround (no way to avoid it).
+A critical issue affects a very large percentage of users (> 60%) and matches at least one of the following:
+
+- It can lead to data loss, introduce a security vulnerability or break the automatic end to end tests
+- It prevents the essential shop operations or puts your business at great risk
+
+Examples:
+
+- Difficulty accessing the front office or back office (significant slowdown, error during installation or update, fatal error)
+- Difficulty to globally manage categories, products or customers
+- Difficulty to globally place and manage orders
+
+A critical issue should result in a patch version that should be released as soon as possible. Our recent [spam attack](http://build.prestashop.com/news/fighting-against-spamming-again/) is a good example: it was a critical security issue as our users were flooded with fake customers’ accounts. As a consequence, a patch version has been released that resolve it on 1.6 and 1.7, a good example of a community work.
+
+
+### Major
+
+The bug affects major functionality or major data and there is a workaround, but it is not obvious or can be difficult to put in practice.
+
+A major issue affects a large percentage of users (> 30%) and matches at least one of the following:
+
+- It impacts law compliance
+- It has a strong impact on the usability of the front-office / back-office or blocks another project
+- It is an important problem but not necessarily blocking the main activity of the seller
+
+Examples:
+
+- Being unable to add, configure or delete a theme or a module
+- Difficulty in operating a module properly
+- Impacts the price the customer pays
+
+### Minor
+
+The bug affects minor functionality or non-critical data and there is a reasonable workaround, even if it can be annoying when using your shop.
+
+
+Examples:
+
+- A tolerable slowdown
+- A display problem that prevents users from doing something non-critical (eg: can’t click on an element that can be accessible in another way)
+- An error message displayed in your back-office that can be dismissed
+- The product doesn't get added to cart after deleting product customisation
```

Comment:
I think we can safely remove this one as the specifics of it are not easy to grasp

Reply:
Done